WORKSPACE

```diff
@@ -53,22 +53,6 @@ yarn_install(
     environment = {
         "SOME_USER_ENV": "yarn is great!",
     },
-    # The @npm//:node_modules_filegroup generated by manual_build_file_contents
-    # is used in the //packages/typescript/test/reference_types_directive:tsconfig_types
-    # test. For now we're still supporting node_modules as a filegroup tho this may
-    # change in the future. The default generated //:node_modules target is a js_library
-    # rule which provides NpmPackageInfo but we have not yet dropped support for
```

Comment:
this looks like good text for the BREAKING CHANGE message
> dropped support for filegroup based node_modules target

Reply:
+1